.github/workflows/ci.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           # GitHub checks PRs out based on the merge commit; we want the branch HEAD.
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0
+      - run: git fetch origin main:main
```

Comment:
Is this needed?

Reply:
Yes, we had to add it to `cbh-core` too.